hs_explore/models.py

```diff
@@ -0,0 +1,242 @@
+from django.db import models, transaction
+from hs_core.models import BaseResource
+from django.contrib.auth.models import User
+from hs_core.hydroshare import user_from_id, get_resource_by_shortkey
+from django.contrib.postgres.fields import HStoreField
+import copy
+
+
+class Status(object):
+    """ define the status of recommended resources and
+        the number for recommended resources made for users
+        STATUS_NEW: a new recommended resource for a user
+        STATUS_SHOWN: a recommended resource has been shown to the user
+        STATUS_EXPLORED: a recommended resource has been explored by the user
+        STATUS_APPROVED: a recommended resource has been approved to be a correct
+        recommendation by the user
+        STATUS_DISMISSED: a recommended resouce is dismissed by the user
```

Comment:
Are we using DISMISSED at all now?

Reply:
The fill_go_stop_words command just does that.

Reply:
DISMISSED is not used for now. It is for future development.